<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>site_recommend</class>
 <widget class="QWidget" name="site_recommend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="locationLists">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>661</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_site">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找景点</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="food_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按美食分数排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sport_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按运动分数排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="trip_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按旅游分数排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="heat_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按热度分数排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="score_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按评分分数排序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="study_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按学习分数排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_page">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_page">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="page_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前页码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入搜索内容</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
